commands/project/create.ts

```diff
@@ -28,36 +37,92 @@ exports.describe = uiBetaTag(i18n(`${i18nKey}.describe`), false);
 exports.handler = async options => {
   const { derivedAccountId } = options;
 
-  const hasCustomTemplateSource = Boolean(options.templateSource);
+  let latestRepoReleaseTag;
```

Comment:
Most of the changes in this file are to make the patterns match the ones in the project add command file. Now both files will:
1. Determine the template source (aka GH repo). This command lets the user override the default
2. Fetch the list of available assets using the config.json file at the repo root
3. Prompt the user for info (name, dest, template)
4. Call `cloneGithubRepo` to download the contents